test/test.js

```diff
@@ -383,43 +383,6 @@ describe('paste-markdown', function () {
       assert.include(textarea.value, tableMarkdown)
     })
 
-    it('pastes markdown with links correctly when identical labels are present', function () {
```

Comment:
removing these tests since we run our tests in ChromeHeadless mode and they should test firefox. The chrome context has been covered by other tests in this file.

Reply:
We can set up the tests to also run in Firefox.